fiddler/manifest.json

```diff
@@ -0,0 +1,43 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "ee617671-508e-4bb3-ba25-8815b11a16aa",
+  "app_id": "fiddler",
+  "classifier_tags": [
+    "Supported OS::Linux",
+    "Supported OS::Mac OS",
+    "Supported OS::Windows"
```

Comment:
```suggestion
    "Supported OS::Windows",
    "Offering::Integration"
```

Reply:
@anawishnoff - why the above change? Does it mean that this integration will not be supported on Mac and Linux?